munin.py

```diff
@@ -323,28 +323,28 @@ def getMISPInfo(hash):
             print("[D] Querying MISP: %s" % m_url)
         try:
             # Preparing API request
-            misp = PyMISP(m_url, m_auth_key, args.verifycert, 'json')
+            misp = pymisp.PyMISP(m_url, m_auth_key, args.verifycert, debug=args.debug, proxies={},cert=None,auth=None,tool='Munin : Online hash checker')
             if args.debug:
                 print("[D] Query: values=%s" % hash)
-            result = misp.search('attributes', values=[hash])
+            result = misp.search('attributes', type_attribute=fetchHash(hash)[1] ,value=hash)
```

Comment:
including type_attribute makes misp server search faster, without this misp gives 500 ( Long execution Times ) trying to search everything in the DB.